lib/ex_auction/auction/worker.ex

```diff
@@ -88,6 +102,7 @@ defmodule ExAuction.Auction.Worker do
       ) do
     case do_place_bid(state, bid) do
       {:ok, %Auction.Bid{} = last_bid} = result ->
+        # TODO: Confirm why we need this sorting, it feels artificial
```

Comment:
We need sorting to make life easy for the strategy modules down the line if they ever need it. English module relies on this assumption currently.